docker/paoSettings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I think this file should not be included since configs might vary from client to client (db connection strings, passwords, encryption keys, etc...)

I provided a valid sample, but I think it should be up to each developer to create their own (even if it's just copying the sample.)

Reply:
The idea here is to provide a docker image that can be used immediately to do development work.  For that, the config is needed.  If a developer wishes to override it, go for it.  But not including it raises the barrier to getting started.

Reply:
I should add, that having this would have made it easy to test my first PR.

Reply:
I see.  That does make sense if it would help get people up and running faster.

I must confess - I'm familiar with docker in general, but not the typical workflow.  If we merge this in, what are the steps to getting this up and running?  Is it really as simple as running those three commands in note.txt?  And is that consistent across OS'es (ie, even on Windows?)

Reply:
Since these are in the container, they should work universally to run the container with no config changes needed to test with the container.